src/react/src/components/DashboardClaimsListTable.jsx

```diff
@@ -19,6 +20,8 @@ import {
     makeProfileRouteLink,
     makeFacilityClaimDetailsLink,
     getIDFromEvent,
+    getComparator,
+    sort,
 } from '../util/util';
 
 const dashboardClaimsListTableStyles = Object.freeze({
```

Comment:
Could you please use classes instead of inline styles? It is a more common way.

Reply:
Done